### DISCUSSION TITLE
Embed not populating via Compose Post link

### DISCUSSION THREAD

**Discussion by atsbob** [bluesky-social/atproto#3072] [Bluesky Client App]
I'm swapping out all my X socials for Bluesky on my website but noticed that the [Compose Post](https://docs.bsky.app/docs/advanced-guides/intent-links) links do not populate embeds when clicked. The pop-up simply shows the title and URL. I thought it was my site at fault but the very demo listed on Bluesky's [documentation](https://docs.bsky.app/docs/advanced-guides/intent-links) does the same.

![image](https://github.com/user-attachments/assets/128c7b03-3afa-4837-a236-b7fcf3b2e4ef)

Only when I manually add a space does Bluesky properly scrape the contents to generate the embed 

![image](https://github.com/user-attachments/assets/dbc80fdb-8495-4e8e-a937-4c0573192be5)

Not sure what information I can provide otherwise. I tried it in both Firefox and Chrome with the same results.

**Comment by SacRebel** [comment]
I came here to see if there was a solution in the works. I create RSS feeds from RSS sites and currently have to post the link and the image from the link to the feed account. The bluesky app is clearly doing something on the inbound side of create post but what??? I also had an issue where if my link text is not the link address when you click a popup warns you are leaving bluesky but if they match it doesn't...I can live with this as a feature but it wasn't clear why it happened initially!

**Reply by atsbob** [reply]
It looks like my reported issue with solved. Maybe through another reported issue as I'm seeing the rich text previews generating now.

**Reply by SacRebel** [reply]
Nice. I was able to get my problem resolved by using the send_ogp_link_card.py example in the python SDK!
